.github/workflows/ci.yml

```diff
@@ -0,0 +1,125 @@
+name: CI
+on:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: pc2fromecs/jdk8hugo
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Why is this step using @v2 when the following steps all use @v3?

Reply:
unclear,  that was copied from icpctools